.idea/membrane-webrtc-ios.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
```

Comment:
don't we need to gitignore this stuff too?

Reply:
done